src/styleguide/_patterns/02-molecules/03-form-group/01-form-group.njk

```diff
@@ -0,0 +1,13 @@
+{%- from 'atoms-label' import label -%}
+{%- from 'atoms-text-input' import textInput -%}
+
+{# Reusable macro for other templates to import: #}
+{%- macro formGroup(form) -%}
+<div class="grav-c-form-group">
+  {{ label(form.label, form.id) }}
```

Comment:
I think it's safer to use `| safe` whenever using macros that output HTML.

While it's configurable, I think Nunjuck's default behaviour is to auto-escape strings. So, if we're not careful this might turn into `&lt;label>` rather than `<label>`.